source/common/quic/platform/flags_impl.cc

```diff
@@ -15,11 +15,12 @@ namespace quiche {
 
 namespace {
 
-absl::flat_hash_map<std::string, Flag*> makeFlagMap() {
-  absl::flat_hash_map<std::string, Flag*> flags;
+absl::flat_hash_map<absl::string_view, Flag*> makeFlagMap() {
+  absl::flat_hash_map<absl::string_view, Flag*> flags;
```

Comment:
Another option to consider is to add QuicFlagPrefix variants to this map in addition to the non QuicFlagPrefix variants during construction.  This simplifies the implementation of findFlag.

Reply:
I think having 2 maps might be an overkill. I created another constant to simplify the flag lookup process.

Reply:
I am suggesting a single map, but include both versions of the name as keys.